ci/README.md

```diff
@@ -0,0 +1,55 @@
+# CI Readme
+
+This project performs integration testing using [Concourse](https://concourse-ci.org/).
+
+## Integration Test Setup
+
+If you want to be able to perform integration tests on your own concourse pipeline you will need
+to create three things:
+
+* A GCP project for the integration tests.
+* A service account in that project that has permissions to read/write/view Pub/Sub topics and
+  subscriptions. **NOTE** this service account should **only** have permissions to do this.
+* A Pub/Sub topic and subscription for testing.
+  These are set to `pubsubbeat-integration-topic` and `pubsubbeat-integration-subscription` in
+  the pipeline configuration by default.
+
+Here is a script that will set up your project:
```

Comment:
Consider pulling both the setup and tear down scripts into their own script files housed within the repo. This part of the doc can then be: Execute setup script: ./ci/setup-int-test.sh
This not only has the benefit of making this doc more concise, but also means less work/moving parts for the user to mess up :)

Reply:
I'd like to leave it as-is for now because copy/paste is more of an at your own risk thing than a shell script so there's less of a risk of a something going wrong and cascading down if you execute one command at a time.

Reply:
Makes sense :)